android/src/main/AndroidManifest.xml

```diff
@@ -37,19 +37,8 @@
       android:exported="false"
       android:enabled="true"
       android:foregroundServiceType="mediaPlayback">
-      <intent-filter>
-        <action android:name="android.media.browse.MediaBrowserService" />
-        <action android:name="android.intent.action.MEDIA_BUTTON" />
-      </intent-filter>
     </service>
 
-    <receiver android:name="androidx.media.session.MediaButtonReceiver"
```

Comment:
how are we going to handle the remote controls on TV or media events firing in the notification player? Should we manage it on the app's end?